app/views/locales/_start_a_chapter_blurb.en.md

```diff
@@ -0,0 +1,8 @@
+# Start a Chapter
+
+The Awesome Foundation isn't actually a foundation at all. Instead, it's a grassroots movement. Our global community is made up of amazing people who have made the decision to make their local communities a bit more awesome. They organize to assemble, launch, and run their own chapters.
+
+<div class="button-wrapper">
+  <a href="/en/start_a_chapter">Get started now!</a>
+</div>
```

Comment:
This button shows up nicely, though we have to have HTML in the markup for it. I wonder if we should also have a button for the `Apply for a Grant` link above on the About Us page.